libs/elasticsearch/langchain_elasticsearch/cache.py

```diff
@@ -17,8 +35,8 @@
 logger = logging.getLogger(__name__)
 
 
-class ElasticsearchCache(BaseCache):
-    """An Elasticsearch cache integration for LLMs."""
+class ElasticsearchCacheIndexer:
+    """Mixin for Elasticsearch clients"""
```

Comment:
So far we avoided mixins as a pattern in this codebase. Would you be okay with sharing the `_manage_index` functionality as a util function?

Reply:
removed the mixin and added those functionalities as utilities. Not sure if they should live there or under client.py. let me know what do you think